labs/CiscoContainerPlatform-101/2.md

```diff
@@ -13,6 +13,39 @@ Notice in the Cluster details page, a VIP has already been assigned to the Maste
 Each Cisco Container platform installation comes with integrated Open API documentation, powered by OpenAPI (Swagger tooling).
 You can access this [HERE (https://10.10.20.110/2/swaggerapi/)](https://10.10.20.110/2/swaggerapi/), where you see all the currently supported platform APIs.
 
+Note: There is a known issue that the Swagger API page may fail to load.
+
+And here is workaround for the issue if you are using Chrome browser:
+
+
+1. Open the swagger UI using the following URL format:```https://<ccp_ui_ip>/2/swaggerapi```
```

Comment:
Open the Swagger API page using the following URL:

Reply:
Updated 2.md